docker.go

```diff
@@ -83,6 +84,35 @@ func (p Plugin) Exec() error {
 		time.Sleep(time.Second * 1)
 	}
 
+	// Create Auth Config File
+	if p.Login.AuthConfig != "" {
+		fmt.Println("AuthConfig provided.")
+		err_mkdir := os.MkdirAll("/root/.docker", 0600)
```

Comment:
The file paths and names should be stored in `daemon.go` and `daemon_win.go` as constants. This will ensure the correct path is for windows. I am not quite sure which path we should use for windows.  cc @donny-dont ?

Reply:
edit: for windows the path is `C:\ProgramData\Docker\config\daemon.json`

Reply:
Ok, but what is the default path for windows?

Reply:
on windows it is `C:\ProgramData\Docker\config\daemon.json`